examples/api/embedding.http

```diff
@@ -50,6 +50,19 @@ Content-Type: application/json
 POST http://localhost:8080/api/v1/embed/3675b69e-8324-4110-bdca-059031aa8da3/get-embed-url
 Content-Type: application/json
 
+// Replace dashboard and project UUIDs with your own
 {
-  "dashboardUuid": "53c634a4-203b-48c3-9336-f6e7c3117b0b"
-}
\ No newline at end of file
+    "user": {
+        "externalId": "steve@here.com"
+    },
+    "content": {
+        "type": "dashboard",
+        "dashboardUuid": "e45dcfb9-d5fe-4c0e-9d8c-70871e904aea",
```

Comment:
projectUuid is consistant, but afaik the dashboard UUID is randomly generated on each /reset-db